_data/rules.yml

```diff
@@ -1546,6 +1546,10 @@ groups:
                 query: 'kube_hpa_status_desired_replicas >= kube_hpa_spec_max_replicas'
                 severity: info
                 for: 2m
+              - name: Kubernetes Pod OOM killed
+                description: Pod {{ $labels.pod }} in namespace {{ $labels.namespace }} is OOMKilled
+                query: kube_pod_container_status_last_terminated_reason{reason="OOMKilled"} > 0
+                severity: critical
```

Comment:
@mverleun consider https://github.com/thaum-xyz/ankhmorpork/blob/master/apps/monitoring/manifests/other/testingPrometheusRule.yaml#L35-L44 as a replacement. It might be the best that is possible now. However ideally, the metric for OOMKills should come from CRI as using kube-state-metrics for this is not really 100% reliable.